observability-lib/dashboards/core-node/component.go

```diff
@@ -373,47 +418,10 @@ func headlines(p *Props) []*grafana.Panel {
 			DisplayMode: common.LegendDisplayModeList,
 			Placement:   common.LegendPlacementRight,
 		},
-		AlertOptions: &grafana.AlertOptions{
-			Summary:     `Uptime less than 90% over last 15 minutes on one component in a Node`,
-			Description: `Component {{ index $labels "service_id" }} uptime in the last 15m is {{ index $values "C" }}%`,
-			RunbookURL:  "https://github.com/smartcontractkit/chainlink-common/tree/main/observability-lib",
-			For:         "15m",
-			Tags: map[string]string{
-				"severity": "warning",
-			},
-			Query: []grafana.RuleQuery{
-				{
-					Expr:       `health{` + p.AlertsFilters + `}`,
-					RefID:      "A",
-					Datasource: p.MetricsDataSource.UID,
-				},
-			},
-			QueryRefCondition: "D",
-			Condition: []grafana.ConditionQuery{
-				{
-					RefID: "B",
-					ReduceExpression: &grafana.ReduceExpression{
-						Expression: "A",
-						Reducer:    expr.TypeReduceReducerMean,
-					},
-				},
-				{
-					RefID: "C",
-					MathExpression: &grafana.MathExpression{
-						Expression: "$B * 100",
-					},
-				},
-				{
-					RefID: "D",
-					ThresholdExpression: &grafana.ThresholdExpression{
-						Expression: "C",
-						ThresholdConditionsOptions: grafana.ThresholdConditionsOption{
-							Params: []float64{90},
-							Type:   grafana.TypeThresholdTypeLt,
-						},
-					},
-				},
-			},
+		AlertsOptions: []grafana.AlertOptions{
+			healthAverageAlertRule(p, 90, map[string]string{"severity": "info"}),
+			healthAverageAlertRule(p, 70, map[string]string{"severity": "warning"}),
+			healthAverageAlertRule(p, 50, map[string]string{"severity": "critical"}),
```

Comment:
So here is that we generate 3 alerts that got different severity depending on the threshold (second param of the function is the threshold)

Reply:
Very necessary, nice